libs/components/indicators/src/lib/modules/illustration/illustration.component.ts

```diff
@@ -26,7 +26,7 @@ const pixelSizes: Record<SkyIllustrationSize, number> = {
  */
 @Component({
   selector: 'sky-illustration',
-  imports: [CommonModule, NgOptimizedImage],
+  imports: [CommonModule],
```

Comment:
Angular was yelling that `NgOptimizedImage` was extra in Angular 19. I just went ahead and removed it.